apps/extension/src/routes/popup/login.tsx

```diff
@@ -0,0 +1,39 @@
+import { Button } from 'ui/components';
+import { FadeTransition, LoginForm } from '../../components';
+import { PagePath } from '../page/paths';
+
+export const Login = () => {
+  const gotoRestorePage = () =>
+    void (async function () {
+      await chrome.tabs.create({
+        url: `${chrome.runtime.getURL('page.html')}#${PagePath.RESTORE_PASSWORD}`,
```

Comment:
Why do we have a hash and not a `/` that goes straight to that route?

Reply:
'chrome-extension://gchndbcadfiekjgpbpjdmipdmmalakmc/page.html/restore-password' like this doesn`t work

Reply:
tab does not change the url, it always looks like this - chrome-extension://gchndbcadfiekjgpbpjdmipdmmalakmc/page.html

Reply:
Ah, yes can you see if the hash router will work?

Reply:
hash router works!